### ISSUE TITLE
How to enable accessibility for Flycut?

### ISSUE THREAD

**Issue by learnwell** [TermiT/Flycut#159]
I updated to osx 10.14 and got a popup asking to give permissions to Flycut when pasting in Mac and I accidentally hit the deny button, how or where should I go to give the appropriate permissions? I cannot paste after selecting the text from Flycut bezel, right now :(

Any hints are appreciated!

**Comment by learnwell** [comment]
* `Security & Privacy` > `Privacy` > `Accessibility` > `Flycut`
* `Security & Privacy` > `Privacy` > `Automation` > `Flycut` > `System Events`

**Comment by mattsoftware** [comment]
How do we enable flycut in accessibility/automation when there is no checkbox there to tick?

**Comment by learnwell** [comment]
> How do we enable flycut in accessibility/automation when there is no checkbox there to tick?

On mac, once you are at `Security & Privacy > Privacy > Accessibility` there should be a little **lock** icon that you can click then it will ask you for password for your account and then let you edit stuff ... click the **plus** icon and add what you need ... I think!?

**Comment by iteniasim** [comment]
or you could just not bother and try to paste with flycut anyway, that way you just have tick the flycut icon in accebility which opens through a prompt asking for accessibility needed to paste

**Comment by amjibaly** [comment]
> or you could just not bother and try to paste with flycut anyway, that way you just have tick the flycut icon in accebility which opens through a prompt asking for accessibility needed to paste

This is the only way I could get Flycut to show up in the accessibility list, just launching it over and over did nothing. Thanks for the tip.
